This is a complicated way to enable features on really old versions of rustc, since we dont use those versions we can just hardcode it to a version > 34 and enable everything.

diff --git a/Cargo.toml b/Cargo.toml
index 498b79f..1545f79 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -18,8 +18,6 @@ build = "build.rs"
 description = "std::error::Error for libcore"
 readme = "README.md"
 license = "MIT OR Apache-2.0"
-[build-dependencies.version_check]
-version = "0.9"
 
 [features]
 alloc = []
diff --git a/build.rs b/build.rs
index 19d1a1e..abea61c 100644
--- a/build.rs
+++ b/build.rs
@@ -1,13 +1,3 @@
-extern crate version_check;
-
 fn main() {
-    let (version, _channel, _date) = version_check::triple().expect("Rustc to give us its version");
-
-    for i in 0..65536 {
-        if version.at_least(&format!("1.{}.0", i)) {
-            println!("cargo:rustc-cfg=rustc_1_{}_0", i)
-        } else {
-            break;
-        }
-    }
+    println!("cargo:rustc-cfg=rustc_1_70_0")
 }
\ No newline at end of file
